sbpy/units/core.py

```diff
@@ -39,6 +27,20 @@
 from ..exceptions import OptionalPackageUnavailable, SbpyWarning
 
 
+__all__ = [
+    'hundred_nm',
+    'spectral_density_vega',
+    'enable',
+    'VEGA',
+    'VEGAmag',
+    'JM',
+    'JMmag',
+    'albedo_unit',
+    'dimensionless_albedo',
+    'projected_size',
+]
+
+
```

Comment:
I've recently learned that, in general, `__all__` should be before import statements.  https://peps.python.org/pep-0008/#module-level-dunder-names

Reply:
Strange.  When I put `__all__` before the import statements, `pycodestyle` check returns errors:

(dev) jyli@adeona code % pycodestyle sbpy/units/core.py
/Users/jyli/.env/dev/lib/python3.8/site-packages/pycodestyle.py:113: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
sbpy/units/core.py:29:1: E402 module level import not at top of file
sbpy/units/core.py:30:1: E402 module level import not at top of file
sbpy/units/core.py:31:1: E402 module level import not at top of file
sbpy/units/core.py:32:1: E402 module level import not at top of file
sbpy/units/core.py:33:1: E402 module level import not at top of file
sbpy/units/core.py:34:1: E402 module level import not at top of file
sbpy/units/core.py:37:1: E402 module level import not at top of file
sbpy/units/core.py:38:1: E402 module level import not at top of file
sbpy/units/core.py:39:1: E402 module level import not at top of file

I'm confused.  Maybe I'll just do `# noqa: E402`?

Reply:
Well that's annoying.  PEP8 has conflicting advice here.  https://peps.python.org/pep-0008/#imports  Then leave it as is.  I'd rather not salt our code with noqa comments.